lib/src/view/settings/settings_screen.dart

```diff
@@ -93,6 +95,10 @@ class _Body extends ConsumerWidget {
       generalPreferencesProvider.select((state) => state.systemColors),
     );
 
+    final locale = ref.watch(
+      generalPreferencesProvider.select((state) => state.locale),
```

Comment:
Same remark as in app.dart file: we have too many `.select`. It is better now to use a single `ref.watch`.